### PR TITLE
Pfappserver Updates

### PR DESCRIPTION
**Pull request by satkunas**
# Description
Continue work on the pfappserver nodes

# Impacts
Increased coffee intake

# Delete branch after merge
YES

### CODE REVIEW

**Comment by julsemaan** [comment]
image

File: html/pfappserver/root/static.alt/src/views/Nodes/_store/index.js [hunk 1]
```diff
@@ -231,6 +231,19 @@ const actions = {
...
   },
+  registerBulkNodes: ({commit}, macs) => {
+    return new Promise((resolve, reject) => {
+      api.registerBulkNodes(macs).then(response => {
+        response.items.filter(item => item.status === 'success').forEach(function (item, index, items) {
+          commit('ITEM_UPDATED', { mac: item.mac, prop: 'status', data: 'reg' })
+        })
+        resolve(response)
+      }).catch(err => {
+        commit('ITEM_ERROR', err.response)
```
Comment (cgx) [inline html/pfappserver/root/static.alt/src/views/Nodes/_store/index.js:243]: This mutation doesn't exist. I guess you wanted to rename `NODE_ERROR`?

File: html/pfappserver/root/static.alt/src/views/Nodes/_store/index.js [hunk 2]
```diff
@@ -244,14 +257,14 @@ const actions = {
     })
   },
   deregisterBulkNodes: ({commit}, macs) => {
-    commit('NODE_REQUEST')
```
Comment (cgx) [inline html/pfappserver/root/static.alt/src/views/Nodes/_store/index.js:247]: Why removed? Should be restored and renamed `ITEM_REQUEST`.

File: html/pfappserver/root/static.alt/src/views/Nodes/_store/index.js (hunk 1 above)
Comment (satkunas) [reply html/pfappserver/root/static.alt/src/views/Nodes/_store/index.js:243]: I wanted to keep NODE_* and ITEM_* seperated, thus I created a ITEM_ERROR instead or reusing NODE_ERROR.

File: html/pfappserver/root/static.alt/src/views/Nodes/_store/index.js (hunk 2 above)
Comment (satkunas) [reply html/pfappserver/root/static.alt/src/views/Nodes/_store/index.js:247]: Renamed to `ITEM_REQUEST` in https://github.com/inverse-inc/packetfence/pull/3273/commits/c5b70c1db1ddf46801049d3d79a699c855f47d80

File: html/pfappserver/root/static.alt/src/views/Nodes/_store/index.js (hunk 1 above)
Comment (cgx) [reply html/pfappserver/root/static.alt/src/views/Nodes/_store/index.js:243]: Got it. Thanks.

**Pull request merged**